api/src/main/java/edu/wpi/first/shuffleboard/api/util/TestUtils.java

```diff
@@ -0,0 +1,30 @@
+package edu.wpi.first.shuffleboard.api.util;
+
+/**
+ * Utility class for making sure certain methods may only be called from tests.
+ */
+public final class TestUtils {
+
+  private TestUtils() {
+    throw new UnsupportedOperationException("TestUtils is a utility class!");
+  }
+
+  /**
+   * Asserts that this method's call stack originated from a test method invocation.
+   *
+   * @throws IllegalStateException if this method is not called from within a test
+   */
+  public static void assertRunningFromTest() throws IllegalStateException {
```

Comment:
Basically just used to make sure the default registries aren't changed during runtime. We really need to get DI set up